lib/event_loop.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const Loopbench = require('loopbench');
+let sampler = null;
+
+
+module.exports = { start, stop, getState };
+
+
+function start (options) {
+  if (sampler) {
+    return;
+  }
+
+  sampler = Loopbench(options);
```

Comment:
should we tune the period at which we check the event loop? Maybe I can support `period` as well in loopbench.